app.js

```diff
@@ -24,15 +24,16 @@ mongoose.connect(process.env.DB_CONNECTION, connectionParams)
         logger.error(error.message);
     })
 
-
 app.use(bodyParser.json())
 
 app.get('/', (req, res) => {
     res.status(200).send('HELLO ˜')
 })
 
+app.use('doc',swaggerUi.serve, swaggerUi.setup(swaggerFile));
+require('./api/routes/backupRouter')(app);

```

Comment:
why do you send the app here?
expres router doesn't require it.

Reply:
I copy paste it from my code in 'communication'.
I didn't touch the routers here yet.
Did you mean it's a wrong way in the 'communication' too?

Reply:
if that is the way you did it, yes its wrong. learn about express router.

Reply:
Ok,
I'll do בעז"ה.
I saw it in example for swagger-autogen, the router exports a function that get the app as input.